src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -34,7 +33,6 @@ public class BindingWorkflowTests
         private ConfigurableSonarQubeServiceWrapper sonarQubeService;
         private ConfigurableVsGeneralOutputWindowPane outputWindowPane;
         private ConfigurableVsProjectSystemHelper projectSystemHelper;
-        private ConfigurableProjectSystemFilter projectFilter;
```

Comment:
The filter is now encapsulated inside the project system and is being updated during connection rather than binding